reinforcement_learning/python-lib/dqn.py

```diff
@@ -0,0 +1,41 @@
+from stable_baselines.common.vec_env import DummyVecEnv
+from stable_baselines.deepq.policies import MlpPolicy
+from stable_baselines.deepq.policies import CnnPolicy
+from stable_baselines import DQN
+  
+def train_dqn(total_timesteps_var,
```

Comment:
remove all these var suffixes, no worries they don't conflict with named arguments, in fact please name them exactly as named arguments, no subtle differences